site/app/templates/grading/electronic/Details.twig

```diff
@@ -595,9 +595,15 @@
 {% endmacro %}
 
 {# Team Name #}
-{% macro render_column_team_name(context, section, graded_gradeable, index, info, column) %}
+{% macro render_column_team_name(context, section, graded_gradeable, index, info, column, max_team_name_length) %}
     <td>
-        {{ graded_gradeable.getSubmitter().getTeam().getTeamName() }}
+        {# existing teams names over max length get truncated down on display #}
+        {% if graded_gradeable.getSubmitter().getTeam().getTeamName()|length > max_team_name_length %}
+          {% set truncatedString = graded_gradeable.getSubmitter().getTeam().getTeamName()|slice(0, max_team_name_length) ~ '...' %}
+          {{ truncatedString }}
```

Comment:
Do we need to do a `set` here, or could this be simply:

```suggestion
          {{ graded_gradeable.getSubmitter().getTeam().getTeamName()|slice(0, max_team_name_length) ~ '...' }}
```

Reply:
sure, why not!
Just thought of breaking it into 2 lines, but never mind.